redis/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM redis:latest
+
```

Comment:
Could you add something about the maintainer? Maybe a generic address related to the team. 😉 

`LABEL maintainer="XXXXX XXXXX <xxxxx@emakina.fr>"`

Reply:
@ajardin fixed